docs/maintenance/ISSUES.md

```diff
@@ -0,0 +1,101 @@
+---
+id: issues
+sidebar_label: Issue Management
+title: Issue Management
+---
+
+This document serves as a guide for how you might manage issues, also known as issue triaging.
+
+Use your best judgement when triaging issues, but most of all remember to be **kind, friendly, and encouraging** when responding to users.
+
+:::tip
+If you're ever unsure on any part of issue management, don't hesitate to loop in a maintainer that has more context to help!
+:::
+
+## Issue Flow
+
+[Issues pending triage](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3Atriage) are given the `triage` label.
+Most issues go through the following triage flow when created:
+
+1. Some user files the issue, and it is given the `triage` label automatically
+2. A maintainer will ensure the issue is valid:
+   - If it doesn't fill out an appropriate template with enough information:
+     - Add the `please fill out the template` and `awaiting response` labels
+     - Ask the poster for more information using a _Needs More Info_ response
+   - If it's a duplicate of an issue that already exists:
+     - Add the `duplicate` label and remove the `bug` label
+     - If it's an obvious duplicate, post a _Clearly Duplicate Issue_ response
+     - If it's not an obvious duplicate, link to the existing issue and explain why
+   - If the code is working as intended:
+     - Add the `working as intended` label and remove the `bug` and `triage` labels
+     - If the behavior is due to the user doing something wrong, such as an incorrect config:
+       - Add the `fix: user error` label
+       - [This issue search has some examples of closing comments](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+sort%3Aupdated-desc+label%3A%22fix%3A+user+error%22+is%3Aclosed)
+     - If the behavior is otherwise expected, [this issue search has some examples of closing comments](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+sort%3Aupdated-desc+label%3A%22working+as+intended%22+-label%3A%22fix%3A+user+error%22+is%3Aclosed+)
```

Comment:
Random comment: Based on my triaging experience, it's very hard to tell which are "working as intended" and which are "user error". I guess it doesn't matter that much?

Reply:
Yeah, I'm finding it hard to think of explicit details on this one. Most of these are from the user doing something wrong that's mentioned in the _Troubleshooting & FAQs_ page _(sigh)_.

Reply:
I categorise them into two sets:
1) if the user has misconfigured something then it's "user error"
2) if the user is reporting correct rule behaviour as a bug then it's "working as intended"